drivers/bridge/setup_ip_tables.go

```diff
@@ -15,6 +15,12 @@ const (
 	IsolationChain = "DOCKER-ISOLATION"
 )
 
+var bridgeChain = []iptables.ChainInfo{
```

Comment:
Please define these inside `removeIPChains()`, I don't think we need them as global vars.